pkgs/applications/window-managers/wayfire/wcm.nix

```diff
@@ -1,16 +1,14 @@
-{ stdenv, lib, fetchFromGitHub, meson, ninja, pkg-config, wayland, wrapGAppsHook
+{ stdenv, lib, fetchurl, meson, ninja, pkg-config, wayland, wrapGAppsHook
 , gnome3, libevdev, libxml2, wayfire, wayland-protocols, wf-config, wf-shell
 }:
 
 stdenv.mkDerivation rec {
   pname = "wcm";
-  version = "0.6.0";
+  version = "0.7.0";
 
-  src = fetchFromGitHub {
-    owner = "WayfireWM";
-    repo = "wcm";
-    rev = "v${version}";
-    sha256 = "1b22gymqfn0c49nf39676q5bj25rxab874iayiq31cmn14r30dyg";
+  src = fetchurl {
+    url = "https://github.com/WayfireWM/wcm/releases/download/v${version}/wcm-${version}.tar.xz";
```

Comment:
Whats the difference between this download and the github repo?

Reply:
I don’t know, but there’d be no need to manually generate and upload a tarball if they didn’t want people to use it.